plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -667,8 +667,69 @@ describe('PersonState.update()', () => {
         expect(hub.personManager.isNewPerson).toHaveBeenCalledTimes(0)
     })
 
+    describe('illegal aliasing', () => {
+        beforeEach(() => {
+            hub.statsd = { increment: jest.fn() } as any
+        })
+
+        it('stops $identify if current distinct_id is illegal', async () => {
+            await personState({
+                event: '$identify',
+                distinct_id: '[object Object]',
+                properties: {
+                    $anon_distinct_id: 'anonymous_id',
+                },
+            }).update()
+
+            const persons = await hub.db.fetchPersons()
+            expect(persons.length).toEqual(1)
+
+            const distinctIds = await hub.db.fetchDistinctIdValues(persons[0])
+            expect(distinctIds).toEqual(expect.arrayContaining(['[object Object]']))
```

Comment:
non-blocking

Whats the reason that the distinctId is set to `[object Object]`? I thought that the fix was meant to prevent this as a distinctId.